tests/test_nautilus_librarian/test_mods/test_libvips/test_domain/test_validate_image_dimensions.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+from nautilus_librarian.mods.libvips.domain.validate_image_dimensions import (
+    file_dimensions_exception,
+    validate_image_dimensions,
+)
+
+
+def it_should_reject_image_bigger_than_expected(libvips_fixtures_dir):
+
+    with pytest.raises(file_dimensions_exception):
+        validate_image_dimensions(libvips_fixtures_dir + "/test_image.tif", 8, 16)
```

Comment:
In general I'm prefering this format to build paths:

```
f"{libvips_fixtures_dir}/test_image.tif"
```

But I think both are OK. Just to let know you. I find it easier to read when you have a long path with a lot of variables.

Reply:
You're right, is more legible. It's just an habit thing.